DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/EditDelegateControllerTests.cs

```diff
@@ -63,18 +63,57 @@ public void Index_returns_not_found_with_delegate_at_different_centre()
             result.Should().BeNotFoundResult();
         }
 
+        [Test]
+        public void Index_shows_centre_specific_email_if_not_null()
+        {
+            // Given
+            const string centreSpecificEmail = "centre@email.com";
+            var delegateUser = UserTestHelper.GetDefaultDelegate(
+                DelegateId,
+                userCentreDetailsId: 1,
+                centreSpecificEmail: centreSpecificEmail,
+                centreSpecificEmailVerified: false
+            );
+            A.CallTo(() => userService.GetDelegateById(DelegateId)).Returns(delegateUser);
+
+            // When
+            var result = controller.Index(DelegateId);
+
+            // Then
+            result.As<ViewResult>().Model.As<EditDelegateViewModel>().CentreSpecificEmail.Should()
+                .Be(centreSpecificEmail);
+        }
+
+        [Test]
+        public void Index_shows_primary_email_if_centre_specific_email_is_null()
+        {
+            // Given
+            var delegateUser = UserTestHelper.GetDefaultDelegate(DelegateId, centreSpecificEmail: null);

```

Comment:
`var delegateUser` is a `Delegate` rather than a `DelegateUser`, so should be renamed